core/io/resource_loader.cpp

```diff
@@ -312,7 +314,9 @@ Ref<Resource> ResourceLoader::_load(const String &p_path, const String &p_origin
 	res_ref_overrides.erase(load_nesting);
 	load_nesting--;
 
-	if (!res.is_null()) {
+	if (res.is_null()) {
+		print_verbose(vformat("Failed loading resource: %s", p_path));
+	} else {
 		return res;
 	}
 
```

Comment:
Alternative, `if (res.is_valid())` to be more positive about the success case.

Reply:
I've applied your suggestion, but using `res.is_valid()` instead, to avoid the double negation. I've also changed a check a few lines above to use `res.is_valid()` as well. There's a benefit in both checks reading the same, as they are, let's say tightly related.